docker/launch-hostagent.sh

```diff
@@ -38,6 +38,83 @@ if [ -w ${PREFIX} ]; then
     mkdir -p ${VARDIR}/lib/aci-containers/k8s-pod-network
 fi
 
+function check_eth {
+    ip link show "$1" | grep -q "$1"
+}
+
+function get_ip {
+    ip addr show "$1" | grep -o 'inet [0-9]\+\.[0-9]\+\.[0-9]\+\.[0-9]\+' | grep -o [0-9].*
+}
+
+function get_mac {
+    ip link show "$1" | awk '/ether/ {print $2}'
+}
+
+# Add host access links
+if check_eth veth_host; then
+    echo "veth_host already exists, skip creation"
+else
+    ip link add veth_host type veth peer name veth_host_ac
+    PEER=$(ip link | grep @veth_host: | awk '{print $2}' | awk -F @ '{print $1}')
+    ip link set $PEER name veth_host_ac
+    ip link set veth_host up
+    ip link set veth_host_ac up
+fi
+ACC_MAC=$(get_mac veth_host)
+
+# FIXME Let deployment decide interface name
+if check_eth eth0; then
+    VTEP_IP=$(get_ip eth0)
+    VTEP_IFACE=eth0
+elif check_eth enp0s8; then
+    VTEP_IP=$(get_ip enp0s8)
+    VTEP_IFACE=enp0s8
+else
+    echo "VTEP interface not found"
+fi
+
+# FIXME make route addition based on pod subnet.
+CHECK=$(ip route show 10.2.56.0/24 | wc -l)
```

Comment:
Is there a plan to fix this hard coded value (here and below)?

Reply:
fixed.